designer/client/src/components/common/ListSeparator.tsx

```diff
@@ -1,7 +1,14 @@
-import { cx } from "@emotion/css";
 import React from "react";
-import styles from "./ListSeparator.styl";
+import styled from "@emotion/styled";
+import { variables } from "../../stylesheets/variables";
+
+const StyledSeparator = styled.hr(({ dark }: { dark?: boolean }) => ({
+    border: "none",
+    borderBottom: "1px solid",
+    margin: "10px 0",
+    color: dark ? variables.panelHeaderBackground : variables.panelTitleTextColor,
+}));
 
 export function ListSeparator(props: { dark?: boolean }): JSX.Element {
```

Comment:
Based on the name of this component, it sounds like an MUI list divider, we can use it in our codebase instead of this component?. https://mui.com/material-ui/react-divider/